org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF

```diff
@@ -14,8 +14,8 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.13.0",
  org.eclipse.xtext.ide,
  org.junit;bundle-version="4.12.0",
- org.eclipse.lsp4j;bundle-version="[0.9.0,0.10.0)";resolution:=optional,
- org.eclipse.lsp4j.jsonrpc;bundle-version="[0.9.0,0.10.0)";resolution:=optional,
```

Comment:
We should keep 0.9.0 as lower bound, don’t you think so?

Reply:
am not sure about this. if so we cant implement any of the new api

Reply:
As long as we don’t implement any of these, there shouldn’t be a problem

Reply:
ok, but when i will live with the deprecations for symbol.depreacted for now
its hard to circumvent them